src/sidebar/components/ToastMessages.tsx

```diff
@@ -16,10 +18,19 @@ function ToastMessages({ toastMessenger }: ToastMessageProps) {
   const messages = store.getToastMessages();
 
   return (
-    <BaseToastMessages
-      messages={messages}
-      onMessageDismiss={(id: string) => toastMessenger.dismiss(id)}
-    />
+    <div
+      className={classnames(
+        // Set an aggressive z-index as we want to ensure toast messages are
+        // rendered above other content
+        'z-10',
+        'absolute left-0 w-full',
```

Comment:
Previous `BaseToastMessages` component had an implicit absolute position. I felt it was better to leave that decision to consumers, so this one needs to be wrapped in an absolute-positioned container.